controllers/managedocs_controller.go

```diff
@@ -67,6 +67,7 @@ const (
 	deployerCSVPrefix            = "ocs-osd-deployer"
 	monLabelKey                  = "app"
 	monLabelValue                = "managed-ocs"
+	sopLink                      = "https://red-hat-storage.github.io/ocs-sop/sop/OSD/{{ .GroupLabels.alertname }}.html"
```

Comment:
I would like for this URL to not be hardcoded, lets read it from an env variable that we will set in the CSV (SOP_ENDPOINT).
This way we will be able to develop and test against a mock website and even differentiate between SOPs for the converged, converged-dev (which will be backed by new and unreviewed SOPs as we continue development) and the upcoming Global Service offering (which will have its own set of SOPs).